src/services/expenses.service.js

```diff
@@ -0,0 +1,86 @@
+let expenses = [];
+let nextId = 1;
+
+const resetExpenses = () => {
+  expenses = [];
+  nextId = 1;
+};
+
+const getExpenses = ({ userId, categoryList, from, to }) => {
+  return expenses.filter((expense) => {
+    let match = true;
+
+    if (userId && expense.userId !== +userId) {
+      match = false;
+    }
+
+    if (from && new Date(expense.spentAt) < new Date(from)) {
+      match = false;
+    }
+
+    if (to && new Date(expense.spentAt) > new Date(to)) {
+      match = false;
+    }
+
+    if (categoryList.length > 0 && !categoryList.includes(expense.category)) {
+      match = false;
+    }
+
+    return match;
+  });
+};
+
+const createExpense = ({ userId, spentAt, title, amount, category, note }) => {
+  const expense = {
+    id: nextId,
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  };
+
+  expenses.push(expense);
+
+  return expense;
```

Comment:
After creating a new expense, ensure to increment `nextId` to maintain unique IDs for future expenses.